Gemfile

```
@@ -39,7 +39,7 @@ gem 'activemodel-serializers-xml', '~> 1.0.1'
 gem 'activerecord-import', '~> 1.5.0'
 gem 'activerecord-session_store', '~> 2.1.0'
 gem 'ox'
-gem 'rails', '~> 7.0', '>= 7.0.3.1'
+gem 'rails', '~> 7.1.2', '>= 7.1.3'
```

Is using this versioning syntax any different than:

```suggestion
gem 'rails', '~> 7.1.3'
```

I understand the current syntax as anything above "7.1.2 and below 7.2" AND "at least 7.1.3" Doesn't the above suggestion say the same a bit more concisely?

---

Yes, it is the same. I updated the rule so we can install any minor release inside 7.1.x that is above 7.1.3.